eblob-bf.spec

```diff
@@ -64,6 +64,18 @@ appending records one after another.
 This package contains libraries, header files and developer documentation
 needed for developing software which uses the eblob library.
 
+%package python
```

Comment:
Why have you separated python?

Eblob is a very low-level library which is unlikely to be used alone, thus it is very convenient to have all needed tools in a single package, thus there is no eblob-devel, eblob-python and other packages.
Also, you have only made Fedora/RHEL package, but we also support Debian based distros.